src/packagedcode/plugin_package.py

```diff
@@ -352,7 +379,7 @@ def get_installed_packages(root_dir, processes=2, **kwargs):
     yield from packages_by_uid.values()
 
 
-def create_package_and_deps(codebase, package_adder=add_to_package, strip_root=False, **kwargs):
+def create_package_and_deps(codebase, package_summary ,package_adder=add_to_package, strip_root=False, **kwargs):
```

Comment:
`package_summary` should have a default value so this function can be used without passing this.